test/distributed/org/apache/cassandra/distributed/test/log/FailedLeaveTest.java

```diff
@@ -28,12 +28,16 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-import harry.core.Configuration;
-import harry.core.Run;
-import harry.model.sut.SystemUnderTest;
-import harry.visitors.GeneratingVisitor;
-import harry.visitors.MutatingRowVisitor;
-import harry.visitors.Visitor;
+import org.apache.cassandra.harry.core.Configuration;
+import org.apache.cassandra.harry.core.Run;
+import org.apache.cassandra.harry.sut.SystemUnderTest;
+import org.apache.cassandra.harry.sut.TokenPlacementModel;
+import org.apache.cassandra.harry.sut.injvm.InJVMTokenAwareVisitExecutor;
+import org.apache.cassandra.harry.sut.injvm.InJvmSut;
+import org.apache.cassandra.harry.sut.injvm.QuiescentLocalStateChecker;
+import org.apache.cassandra.harry.visitors.GeneratingVisitor;
+import org.apache.cassandra.harry.visitors.MutatingRowVisitor;
+import org.apache.cassandra.harry.visitors.Visitor;
```

Comment:
nit: Move these imports down to where `HarryHelper` sits

Reply:
weird my IDE did that!